tests/openvino/test_quantization.py

```diff
@@ -64,8 +64,8 @@ def get_num_quantized_nodes(ov_model):
 class OVQuantizerTest(unittest.TestCase):
     # TODO : add models
     SUPPORTED_ARCHITECTURES_WITH_EXPECTED_QUANTIZED_MATMULS = (
-        (OVModelForSequenceClassification, "hf-internal-testing/tiny-random-bert", 42, 32),
-        (OVModelForCausalLM, "hf-internal-testing/tiny-random-gpt2", 41, 21),
+        (OVModelForSequenceClassification, "hf-internal-testing/tiny-random-bert", 32, 35),
+        (OVModelForCausalLM, "hf-internal-testing/tiny-random-gpt2", 41, 22),
```

Comment:
Is this due to the move from ONNX export path to direct nn.Module -> OV export path or something else?

Reply:
changes in openvino and nncf